birdhouse/pavics-compose.sh

```diff
@@ -115,9 +118,14 @@ if [ x"$1" = x"up" ]; then
   done
 fi
 
+log INFO "Executing docker-compose with extra options: ${COMPOSE_EXTRA_OPTS} $*"
 # the PROXY_SECURE_PORT is a little trick to make the compose file invalid without the usage of this wrapper script
-PROXY_SECURE_PORT=443 HOSTNAME=${PAVICS_FQDN} docker-compose ${COMPOSE_CONF_LIST} $*
+PROXY_SECURE_PORT=443 HOSTNAME=${PAVICS_FQDN} docker-compose ${COMPOSE_CONF_LIST} ${COMPOSE_EXTRA_OPTS} $*
```

Comment:
won't this result in (for example):

```
docker-compose --remove-orphans up -d
```

I think this needs to be changed to reverse the order of `$*` and `${COMPOSE_EXTRA_OPTS}`